xet-tools.rb

```diff
@@ -16,8 +16,12 @@ class XetTools < Formula
   end
 
   def install
-    bin.install 'xet'
+    # bin.install 'xet'
     bin.install 'git-xet'
+    bin.echo 'Update your repo config to finish installation:
```

Comment:
This works! Also, homebrew has [a caveats block](https://github.com/xetdata/xethub/pull/new/brian/cas-staging-support) where this could also go.